test_runner/fixtures/neon_fixtures.py

```diff
@@ -1470,6 +1505,33 @@ class ComputeCtl(AbstractNeonCli):
     COMMAND = "compute_ctl"
 
 
+class NeonAttachmentService:
+    def __init__(self, env: NeonEnv):
+        self.env = env
+
+    def start(self):
+        self.env.neon_cli.attachment_service_start()
+        self.running = True
+        return self
```

Comment:
Avoid future test bugs by asserting it's not currently running.

```suggestion
    def start(self):
        assert not self.running
        self.env.neon_cli.attachment_service_start()
        self.running = True
        return self
```